lib/modules/platform/bitbucket/readme.md

```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
+
+For most users, the easiest way to get started is to install [the Mend app for Bitbucket](https://marketplace.atlassian.com/apps/1232072/mend?tab=overview&hosting=cloud) and use the free Renovate plan.
```

Comment:
[Our style guide](https://github.com/renovatebot/renovate/blob/main/docs/development/style-guide.md) says:

> ## Correct name for the GitHub app
>
> Refer to the GitHub app version of Renovate as "the Mend Renovate app".

I don't know if you still want to follow this style? Your review suggestions all boiled down to `the app`.

Just wanted to check that the review suggestions I just applied are what you really want. :wink:

Reply:
The Bitbucket Cloud app is a new/different one and not Renovate-specific. The existing guide still is correct for github